network.sh

```diff
@@ -193,31 +196,94 @@ if [ -n "${NETWORK_CONFIG_FOLDER:-}" ]; then
   NETWORK_CONFIG_FOLDER="$(readlink -m $NETWORK_CONFIG_FOLDER)"
 fi
 
-function set_api_and_ingress_vip() {
+function is_varset_add_comma() {
+    # Description:
+    #     Adds a comma between both given parameters. If only one parameter is
+    #     given, no comma is added and only the given parameter printed. Useful
+    #     for multiple API_VIPs or INGRESS_VIPs Example:
+    #     192.168.111.5,fd2e:6f44:5dd8:c956::14
+    #
+    # Returns:
+    #     Print the string using comma
+
+    # if string is *NOT* NULL/empty
+    if [[ $# -ge 2 ]]; then
+        echo "$1,$2"
+    else
+        echo "$1"
+    fi
+}
+
+function get_vips() {
+    # Arguments:
+    #     None
+    #
+    # Description:
+    #     Gets the INGRESS VIP and API VIP addresses (ipv4 and ipv6)
+    #
+    # Returns:
+    #     None
+    #
+    if [[ -n "${EXTERNAL_SUBNET_V4}" ]]; then
+        API_VIPs=$(dig +noall +answer "api.${CLUSTER_DOMAIN}" @$(network_ip ${BAREMETAL_NETWORK_NAME}) | awk '{print $NF}')
```

Comment:
nit: I don't really like using lower-case letters as part of an upper-case variable name